Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "pixels"
 description = "A tiny library providing a GPU-powered pixel frame buffer."
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
I usually don't bump the version number until I am ready for a release. But I also know that this can lead to confusion. Just FYI.